react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -15,94 +16,82 @@ import { updateDialInNumbers } from '../actions';
 import { InfoDialog } from './info-dialog';
 
 /**
- * The amount of time, in milliseconds, to wait until automatically showing
- * the {@code InfoDialog}. This is essentially a hack as automatic showing
- * should happen in a lonely call and some time is needed to populate
- * participants already in the call.
+ * The type of the React {@code Component} props of {@link InfoDialogButton}.
  */
-const INFO_DIALOG_AUTO_SHOW_TIMEOUT = 1500;
+type Props = {
+
+    /**
+     * The redux state representing the dial-in numbers feature.
+     */
+    _dialIn: Object,
+
+    /**
+     * Whether or not the {@code InfoDialog} should display automatically when
+     * in a lonely call.
+     */
+    _disableAutoShow: boolean,
+
+    /**
+     * Whether or not the local participant has joined a
+     * {@code JitsiConference}. Used to trigger auto showing of the
+     * {@code InfoDialog}.
+     */
+    _isConferenceJoined: Boolean,
+
+    /**
+     * The URL for a currently active live broadcast
+     */
+    _liveStreamViewURL: string,
+
+    /**
+     * The number of real participants in the call. If in a lonely call, the
+     * {@code InfoDialog} will be automatically shown.
+     */
+    _participantCount: number,
+
+    /**
+     * Whether or not the toolbox, in which this component exists, is visible.
+     */
+    _toolboxVisible: boolean,
+
+    /**
+     * Invoked to toggle display of the info dialog.
+     */
+    dispatch: Function,
```

Comment:
It's totally unrelated to this PR, just saw this line and made me wonder: Maybe we should use the Dispatch<*> type in the props too and that might help us avoid some flow type problems like we had yesterday? Don't change it, just to kick off this thinking